### PR TITLE
CARMA-WEB-UI: traffic signal intersection id and group id are not available

### PR DESCRIPTION
**Pull request by aswath1**
# PR Details
## Description
There is one traffic signal controller respectively at each side of the road to signalize the vehicle whether the vehicle should stop at the intersection or not. In another word, there are two traffic signal controllers in total for the workzone use case.

The traffic signal controllers are constantly sending signal to v2xhub. V2xhub forward the corresponding SPAT messages to the SV vehicle. However, vehicle OBU receives all the SPAT messages from both traffic signal controllers. The Platform and UI need to process/display the SPAT correctly and distinguish which SPAT is from the approaching traffic signal controller for the SV.

## Related Issue

https://github.com/usdot-fhwa-stol/carma-platform/issues/1420
## Types of changes

- [x] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [x] I have added any new packages to the sonar-scanner.properties file
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) 
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: carma_wm_ctrl/include/carma_wm_ctrl/WMBroadcaster.h [hunk 1]
```diff
@@ -283,8 +297,7 @@ class WMBroadcaster
                                      Each should have gf_pts, affected_parts.
    * \param parallel_llt_front A lanelet whose end should connect to front diagonal lanelet
    * \param parallel_llt_back A lanelet whose start should connect to back diagonal lanelet
-   * \param middle_opposite_lanelets A lanelet list whose front() connects to front diagonal, 
-   *                                 back() connects to back diagonal (their directions are expected to be opposite of parallel ones)
+   * \param middle_opposite_lanelets A lgetInterGroupIdsByLightRegIdk() connects to back diagonal (their directions are expected to be opposite of parallel ones)
```
Comment (MishkaMN) [inline carma_wm_ctrl/include/carma_wm_ctrl/WMBroadcaster.h:300]: extra comments here

**Review by MishkaMN — CHANGES_REQUESTED** [review]
overall looks good to me except typo. It includes some extra changes from develop for participant type I think. I saw release:a1 config/VehicleConfigParams still supports it, so it wouldn't be too bad to have them?

File: carma_wm_ctrl/include/carma_wm_ctrl/WMBroadcaster.h [hunk 2]
```diff
@@ -168,6 +170,21 @@ class WMBroadcaster
...
+ * @param participant vehicle participation type
+ */
+  void setVehicleParticipationType(std::string participant);
+
+  /**
+   * @brief Get the Vehicle Participation Type object
+   * 
+   * @return Current Vehicle Participation Type being used in this World Model Instance
+   */
+  std::string getVehicleParticipationType();
```
Comment (msmcconnell) [inline carma_wm_ctrl/include/carma_wm_ctrl/WMBroadcaster.h:186]: Looks like this still contains Isaiah's changes to develop from a few days ago

**Review by msmcconnell — CHANGES_REQUESTED** [review]

**Review by msmcconnell — APPROVED** [review]
LGTM. Please test before merging

**Review by MishkaMN — APPROVED** [review]

**Comment by MishkaMN** [comment]
tested on fusion at TFHRC

**Pull request merged**